deploy/kubernetes/base/node_linux/node.yaml

```diff
@@ -38,13 +38,13 @@ spec:
               mountPath: /csi
             - name: registration-dir
               mountPath: /registration
-          livenessProbe:
-            initialDelaySeconds: 3
-            exec:
-              command:
-                - /csi-node-driver-registrar
-                - --kubelet-registration-path=/var/lib/kubelet/plugins/pd.csi.storage.gke.io/csi.sock
-                - --mode=kubelet-registration-probe
+        - name: liveness-probe
```

Comment:
This needs an (http) liveness probe on the driver. However, since the pod is running hostNetwork, that consumes a host port which I don't think we want to do by default.